EXERCISE-4.md

```diff
@@ -242,6 +282,185 @@ counters.
 In this case, you should see approx 1 packet/s, as that's the rate of
 packet-outs generated by the `lldpprovider` app.
 
+## Host discovery & L2 bridging
+
```

Comment:
Add some context:
- We enabled link discovery with packetio, now host discovery should work too